docs/guides/cache/index.md

```diff
@@ -1,87 +1,179 @@
+  --cache-to type=registry,ref=<user>/<cache-image>,mode=max \
+  --cache-from type=registry,ref=<user>/<cache-image>
 ```
+
+This option is only set when exporting a cache, using `--cache-to`. When
+importing a cache (`--cache-from`) the relevant parameters are automatically
+detected.
+
+In `min` cache mode (the default), only layers that are exported into the
+resulting image are cached, while in `max` cache mode, all layers are cached,
+even those of intermediate steps.
+
+While `min` cache is typically smaller (which speeds up import/export times, and
+reduces storage costs), `max` cache is more likely to get more cache hits.
+Depending on the complexity and location of your build, you should experiment
+with both parameters to get the results.
+
+### Cache compression
+
+Since `registry` cache image is a separate export artifact from the main build
+result, you can specify separate compression parameters for it. These parameters
+are similar to the options provided by the `image` exporter. While the default
+values provide a good out-of-the-box experience, you may wish to tweak the
+parameters to optimize for storage vs compute costs.
+
+To select the compression algorithm, you can use the
+`compression=<uncompressed|gzip|estargz|zstd>` option. For example, to build the
+cache with `compression=zstd`:
+
+```console
+$ docker buildx build . --push -t <user>/<image> \
+  --cache-to type=registry,ref=<user>/<cache-image>,compression=zstd \
+  --cache-from type=registry,ref=<user>/<cache-image>
+```
+
+Use the `compression-level=<value>` option alongside the `compression` parameter
+to choose a compression level for the algorithms which support it:
+
+- 0-9 for `gzip` and `estargz`
+- 0-22 for `zstd`
+
+As a general rule, the higher the number, the smaller the resulting file will
+be, and the longer the compression will take to run.
+
+Use the `force-compression=true` option to force re-compressing layers imported
+from a previous cache, if the requested compression algorithm is different from
+the previous compression algorithm.
+
+> **Note**
+>
+> The `gzip` and `estargz` compression methods use the
+> [`compress/gzip` package](https://pkg.go.dev/compress/gzip), while `zstd` uses
+> the
+> [`github.com/klauspost/compress/zstd` package](https://github.com/klauspost/compress/tree/master/zstd).
+
+### OCI media types
+
+Like the `image` exporter, the `registry` cache exporter supports creating
+images with Docker media types or with OCI media types. To enable OCI Media
+types, you can use the `oci-mediatypes` property:
+
+```console
+$ docker buildx build . --push -t <user>/<image> \
+  --cache-to type=registry,ref=<user>/<cache-image>,oci-mediatypes=true \
+  --cache-from type=registry,ref=<user>/<cache-image>
+```
+
+This property is only meaningful with the `--cache-to` flag. When fetching
+cache, BuildKit will auto-detect the correct media types to use.
+
+<!-- FIXME: link to image exporter guide when it's written -->
```

Comment:
FIXME should be at the top of the configuration options section (since most of the parameters overlap).